Projects/Foundations/Schemas/Docs-Topics/P/Pluvtech/Pluvtech-Data-Mine/pluvtech-data-mine-002-fisher-transform.json

```diff
@@ -4,70 +4,140 @@
+            "translations": [
+                {
+                    "language": "RU",
+                    "text": "Длину расчета преобразования Фишера можно изменить, найдя и открыв Javascript Code в разделе Data Building Procedure -> Procedure Initialization под продуктом 'Fisher Transform' Product Definition.",
+                    "updated": 1645609781306
+                }
+            ]
         },
         {
             "style": "Png",
             "text": "PNGs/Foundations/Docs/indicators/Pluvtech-Fisher-params.png",
             "updated": 1611673329568
         },
-		{
+        {
             "style": "Title",
-            "text": "Products & Properties"
+            "text": "Products & Properties",
+            "translations": [
+                {
+                    "language": "RU",
+                    "text": "Продукты и свойства",
+                    "updated": 1645609786700
+                }
+            ]
         },
         {
             "style": "Text",
-            "text": "The following properties are available to access:"
+            "text": "The following properties are available to access:",
+            "translations": [
+                {
+                    "language": "RU",
+                    "text": "Доступны следующие свойства:",
+                    "updated": 1645609795155
+                }
+            ]
         },
         {
             "style": "Table",
             "text": "| Product Name | Product Variable | Properties |\n| Fisher Transform | Fisher | fisher, trigger |",
             "updated": 1636853769
         },
-		{
+        {
             "style": "Text",
             "text": "Example:",
-            "updated": 1611661674316
+            "updated": 1611661674316,
+            "translations": [
+                {
+                    "language": "RU",
+                    "text": "Пример:",
+                    "updated": 1645609803924
+                }
+            ]
         },
-		{
+        {
             "style": "Text",
-            "text": "The Fisher transform could be used to generate a buy signal when the Fisher line crosses the trigger line after an extreme low."
+            "text": "The Fisher transform could be used to generate a buy signal when the Fisher line crosses the trigger line after an extreme low.",
+            "translations": [
+                {
+                    "language": "RU",
+                    "text": "Преобразование Фишера можно использовать для генерирования сигнала на покупку, когда линия Фишера пересекает линию триггера после экстремального минимума.",
+                    "updated": 1645609814009
+                }
+            ]
         },
-		{
+        {
             "style": "Javascript",
             "text": "chart.at04hs.Fisher.fisher > chart.at04hs.Fisher.trigger && chart.at04hs.Fisher.fisher < -2",
             "updated": 1636853769
         },
-		{
+        {
             "style": "Text",
```

Comment:
Did you find those small mistakes when translating or saw those random. Either way, great that you find and fix them :+1: